pyrasterframes/src/main/python/requirements.txt

```diff
@@ -10,3 +10,5 @@ geopandas>=0.6.2,<0.7
 descartes>=1.1.0,<1.2
 pytz
 matplotlib
+rtree
+Pillow
```

Comment:
What about `setup.py`?

Reply:
Hmm. We are only using `pillow` in the docs sources. So that's a judgement call. The requirements.txt is explictly used as part of the docs build CI process. 

`rtree` changes were a merge conflict.... Not sure where thats going to land

Reply:
we could possibly trim down the `setup_requires` list by relying on the `requirements.txt` to define the dependencies for building docs. And carefully check that `tests_require` is correct. So, with the exception of `pweave` which we depend on in the definition of the doc build tasks, the following are declared as `setup_requires` but really could be removed?

* rasterio: used in docs and test, but not required for running setup
* folium: only docs; we carefully avoid depending on it in rf_ipython; will enhance rendering of folium.Map if library is available
* boto3: suspect this is related to rasterio
* geopandas: test and docs only
* requests: docs only